Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,17 @@
-import React from 'react';
-import './Header.css';
+import React from "react";
+import "./Header.css";
+
+const HeaderContent = () => {
+  return (
+    <div className="header-content">
+      <p>
+        Let's learn React by building simple interfaces with components. Don't
+        try to overthink it, just keep it simple and have fun. Once you feel
+        comfortable using components you are well on your way to mastering
+        React!
+      </p>
+    </div>
```

Comment:
you can apply `header-content` to the `p` tag here instead and remove the outer div